source/utils/ConfigManager.h

```diff
@@ -26,12 +26,21 @@
 class CreatureDefinition;
 class Weapon;
 class SpawnCondition;
+class CreatureMood;
 
 //! \brief This class is used to manage global configuration such as network configuration, global creature stats, ...
 //! It should NOT be used to load level specific stuff. For that, there if GameMap.
 class ConfigManager : public Ogre::Singleton<ConfigManager>
 {
 public:
+    enum CreatureMoodLevel
+    {
+        Happy,
+        Neutral,
+        Upset,
+        Angry,
+        Furious
+    };
```

Comment:
> I would also say that we should add in the coding good practices to use enum classes for enums (and start replacing the unscoped enumerations currently in the code)

We can, but there is no urgency. Let's switch "when we see one in our way" unless you plan to focus on that in a particular commit, of course. :)

Btw, with mood, and hopefully soon, workshop, new forge, and normals support. 0.5.0 will truly shine. :D